apps/src/studio/studio.js

```diff
@@ -1546,6 +1546,15 @@ Studio.execute = function() {
   Studio.waitingForReport = false;
   Studio.response = null;
 
+  var usesFunctionalBlocks = level.useContractEditor;
```

Comment:
Can you think of situations where this check might not actually be the right one?